clouddriver-titus/src/main/groovy/com/netflix/spinnaker/clouddriver/titus/deploy/handlers/TitusExceptionHandler.java

```diff
@@ -46,10 +46,23 @@ private boolean isRetryable(StatusRuntimeException statusRuntimeException) {
       case UNKNOWN:
         return true;
       case INVALID_ARGUMENT:
-        String message = statusRuntimeException.getMessage();
-        return message != null && message.toLowerCase().contains("rate exceeded");
+        return invalidArgumentConditional(statusRuntimeException.getMessage());
       default:
         return false;
     }
   }
+
+  private boolean invalidArgumentConditional(String statusRuntimeExceptionMessage) {
+    if (statusRuntimeExceptionMessage == null) {
+      return false;
+    }
+
+    boolean rateExceeded = statusRuntimeExceptionMessage.toLowerCase().contains("rate exceeded");
+    boolean assumeRoleError =
+        statusRuntimeExceptionMessage
+            .toLowerCase()
+            .contains("field: 'jobiamvalidator', description: 'titus cannot assume into role");
```

Comment:
Might be a little specific and subject to some structural change/ordering of attributes ... ?

```
contains("jobiamvalidator") && contains("titus cannot assume into role")
```